guide/miniscript-wallet-simple.md

```diff
@@ -0,0 +1,366 @@
+
+{% include image-gallery.html pages = page.images_backup %}
+
+The backup file does not contain any private key data. This means that it is ok to store it digitally in a password manager, for example. However, since it contains all extended public keys, anybody who finds this file can potentially monitor all transactions in real time.
+
+Because they might not want to backup the wallet right away, users will find the backup file and other important information on the wallet settings page at any time. 
+
+
+### Co-signer onboarding
+
+The next step the co-signers set up the wallet on their end. To make this experience easy and seamless, the application allows Bob to invite Alice in a variety of ways:
+
+1. By displaying a QR code that Alice scans with her app.
+2. By sending an invite, which contains the wallet descriptor, over email, Nostr, or any messenger app.  
+3. By providing Alice with a copy of the wallet backup file that he saved previously.
+
+{% include image-gallery.html pages = page.images_cosigner-onboarding %}
+
+
+### Spending from the wallet
+
+Creating transactions and spending from the wallet works like with any other multi-key wallet. We have covered how this works on the [savings wallet reference design](https://bitcoin.design/guide/savings-wallet/#making-small-payments), if you would like to learn more.  
+
+### Managing the recovery path
+
+As mentioned above, recovery paths are powered by timelocks. Technically, however, these timelocks are not applied at the wallet level but at the [UTXO](https://bitcoin.design/guide/glossary/#unspent-transaction-output-utxo) level. This means that every UTXO has its own timelock, based on the time when it was deposited into the wallet. 
+
+To prevent the recovery path from kicking in and refreshing the timelock, a UTXO has to be spent and re-deposited back into the wallet. Please refer to the [custom spending conditions page](https://bitcoin.design/guide/how-it-works/custom-spending-conditions/) for more information about how timelocks work. 
+ 
+
+##### Preventing recovery path activation
+
+So if Bob has three UTXOs in his wallet which he received at different times, he actually needs to refresh three different timelocks. This can quickly become cumbersome and costly, as every refresh transaction implies on-chain transaction fees.
+
+To reduce this complexity, our application offers a "unified" timelock reset experience by default. This simply means that it uses the UTXO that is closest to being unlocked as the trigger to start the process. In the background, however, all UTXOs are automatically batched together. 
+
+This allows us to show one timelock, which makes it easier to understand for users and is more in line with their expectations. An added benefit of this approach is that, as a result, Bob has only one UTXO in his wallet after the timelock refresh.
+
+{% include image-gallery.html pages = page.images_prevent-unlock %}
```

Comment:
In the second image, I don't think you need to show the "2-of-3" and "Recovery path enabled" info in the card at the top. 

In the third image, could this be simplified somehow? Default could be to refresh all at once, which should be more efficient over time due to UTXO consolidation. Then users could change that to pick "only upcoming ones". The list of individual coins with dates seem like something that should be one level further down.

I see you like to put borders around related lines of information for visual grouping. They are barely noticeable due to the low contrast, so I'm not convinced they work as is. You're also losing space and then need to reduce the font size. Maybe keep the slightly larger font for easier readability and use a different treatment, like a light background 
color behind the box that stretches to the edges?
In screen 5, Bob may also want to sign himself with the second hardware wallet.

Reply:
> In the second image, I don't think you need to show the "2-of-3" and "Recovery path enabled" info in the card at the top.

Removed. Also on the wallet detail screens.

> I see you like to put borders around related lines of information for visual grouping. They are barely noticeable due to the low contrast, so I'm not convinced they work as is. You're also losing space and then need to reduce the font size. Maybe keep the slightly larger font for easier readability and use a different treatment, like a light background. color behind the box that stretches to the edges?

I updated all screens (I hope) to use larger fonts in most places and removed the the lines. 

> In screen 5, Bob may also want to sign himself with the second hardware wallet.

Updated the screen flows to account for this.

Reply:
> In the third image, could this be simplified somehow?

We could do something more radical, like in the attached mockup. Where you have just a countdown. Via the advanced settings button users would access the details and UTXO list. 
![alternative](https://github.com/BitcoinDesign/Guide/assets/112604177/9d6ae704-89a7-4d0c-8214-a3acf5b8d367)